pkgs/aimstack/ml/adapters/keras.py

```diff
@@ -84,7 +67,6 @@ def metrics(
 
     def close(self) -> None:
         if self._run:
-            self._run.close()
```

Comment:
Why this line is removed? Will it be enough to just delet or set the run instance to None?